Elevel/Application/Features/QuestionCommands/DeleteQuestionCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using AutoMapper;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.QuestionCommands
+{
+    public class DeleteQuestionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid? Id { get; set; }
```

Comment:
Do you need all these properties for delete (15-19)? I think it could be simplified.

Reply:
Every property except for Id is deleted, resolved

Reply:
Id should be not null, what the sense to try to delete question with null id? when it is not null, so it will return BadRequest at start of processing your request.

Reply:
non-nullified after additional editing. I don't know how could I miss that, actually